articles.tex

```diff
@@ -98,9 +98,9 @@
 A quorum of fifty percent of the Total Number of Possible Votes is required for passage.
 
 \asubsection{Semantic Changes}
-Any semantic change to the Constitution requires the change to be proposed in writing for discussion at a House Meeting.
+Any semantic change to the Constitution requires the change to be proposed in writing for a History Disussion meeting.
```

Comment:
This is actually different than my original intentions of it being proposed to just the History Discussion meeting itself, but I can see concerns with people not even knowing about it to attend the meeting. In my head I was imagining it working like how current discussion topics are announced. I'll have to figure out the best way to word this though, probably say:
```
...requires the change to be proposed in writing at a House Meeting.
The amendment will be discussed at the following History Discussion meeting.
```
or something like that depending on what people like.